README.md

```diff
@@ -97,6 +97,8 @@ root. Available configuration options:
  * `cache` default `false` - set to `true` to enable caching on Google Cloud using datastore
  * `debug` default `false` - set to `true` to log console messages from within the
  rendered pages.
+ * `renderOnly` - restrict the endpoint to only service requests for certain domains. Specified
```

Comment:
Don't fully understand your comment. Are you saying it should only be a domain match?

I wanted to keep it simple since you may want to restrict to a series of sub domains etc. So if there are multiple protocols, they should specify them individually. Updated the readme.